README.md

```diff
@@ -225,6 +225,8 @@ This mode will allow multiple instances of Portainer to connect to a single agen
 
 Note: Due to the fact that the agent will now decode and parse the public key associated to each request, this mode might be less performant than the default mode.
 
+If `AGENT_SECRET` isn't supplied, the agent will turn off after 3 days if not associated to any Portainer instance. This duration can be changed by supplying `AGENT_SECURITY_SHUTDOWN` environment variable in the format "3d5h2s"
```

Comment:
time.Durtion doesn't support days, hours is the max measure: https://pkg.go.dev/time#ParseDuration

Reply:
fixed and added a link